From 297a29aa8a27e93b2fa1fdc40a4964f0fbbaa4ec Mon Sep 17 00:00:00 2001
From: "Min Hu (Connor)" <humin29@huawei.com>
Date: Thu, 31 Mar 2022 14:20:42 +0800
Subject: [PATCH 11/13] net/hns3: change dump file name

change dump file name 'hns3_ethdev_dump.c' to 'hns3_dump.c', as it
is more simple.

Signed-off-by: Min Hu (Connor) <humin29@huawei.com>
---
 drivers/net/hns3/{hns3_ethdev_dump.c => hns3_dump.c} | 0
 drivers/net/hns3/meson.build                         | 2 +-
 2 files changed, 1 insertion(+), 1 deletion(-)
 rename drivers/net/hns3/{hns3_ethdev_dump.c => hns3_dump.c} (100%)

diff --git a/drivers/net/hns3/hns3_ethdev_dump.c b/drivers/net/hns3/hns3_dump.c
similarity index 100%
rename from drivers/net/hns3/hns3_ethdev_dump.c
rename to drivers/net/hns3/hns3_dump.c
diff --git a/drivers/net/hns3/meson.build b/drivers/net/hns3/meson.build
index 665b2afedf..9f30f6b7af 100644
--- a/drivers/net/hns3/meson.build
+++ b/drivers/net/hns3/meson.build
@@ -30,7 +30,7 @@ sources = files(
         'hns3_tm.c',
         'hns3_ptp.c',
         'hns3_common.c',
-	'hns3_ethdev_dump.c',
+	'hns3_dump.c',
 )
 
 deps += ['hash']
-- 
2.30.0

